src/sql/src/statements.rs

```diff
@@ -260,10 +260,12 @@ pub fn column_def_to_schema(column_def: &ColumnDef, is_time_index: bool) -> Resu
 pub fn sql_column_def_to_grpc_column_def(col: ColumnDef) -> Result<api::v1::ColumnDef> {
     let name = col.name.value.clone();
     let data_type = sql_data_type_to_concrete_data_type(&col.data_type)?;
-    let nullable = col
-        .options
-        .iter()
-        .any(|o| matches!(o.option, ColumnOption::Null));
+
+    let nullable = col.options.is_empty()
```

Comment:
Nullable as the default means we can't find `ColumnOption::NotNull` in `col.options`. This modification is not enough here.

Reply:
it's more clear, updated.

Reply:
> Nullable as the default means we can't find `ColumnOption::NotNull` in `col.options`. This modification is not enough here.

Cartoons